package.json

```diff
@@ -9,12 +9,12 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "grunt": "0.4.5",
-    "grunt-cli": "^1.2.0",
-    "grunt-parallel": "^0.4.1",
-    "grunt-shell": "^1.1.2",
-    "jasmine-node": "^1.14.5",
-    "saucelabs": "^1.0.1",
-    "selenium-webdriver": "2.48.0"
+    "grunt": "1.0.4",
```

Comment:
I personally hate grunt and gulp because they are extra overhead, are they really needed in this project?
We can leave it in (I don't have a different solution for now), but it would be nice to remove it in the future

Reply:
I agree. I considered just re-writing it with jest but I think that should be it's own project on it's own repo and we can then deprecate this one.

Reply:
I would not deprecate this one, but keep it. By the way, jest is not a replacement for Grunt/Gulp, it's like Jasmine/Mocha